src/components/Counter/Coounter.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './Counter.scss';
+
+export class Counter extends React.Component {
+  state = {
+    count: 0,
+  }
+
+  add = (addition) => {
+    this.setState(prevState => ({
+      count: prevState.count + addition,
+    }));
+  }
+
+  // sorry, but I have NO idea how to call this method
+  abraCadabra = () => {
+    this.add(1);
+    if (!(this.state.count % 5)) {
+      this.add(100);
+    }
```

Comment:
```suggestion
      if (!(this.state.count + 1 % 5)) {
      this.add(101);
    } else {
      this.add(1);
    }
```
так ти викликатимеш setState тільки 1 раз у випадку якщо в If буде true

Reply:
і додавати 101 потрібно коли ділиться на 5 число після додавання 1 , але це не критично